### ISSUE TITLE
Freeling

### ISSUE THREAD

**Issue by jakoble** [clarin-eric/resource-families-issues#302]
http://hdl.handle.net/20.500.11752/ILC-72

- [ ] Missing input and output info
- [ ] Missing licence info

**Comment by francescafrontini** [comment]
For the license:  CLARIN-DSPACE does not seem to allow you to insert a license when there is no deposit, at least from the web form. We checked and this issue is common to other repositories using CLARIN-DSPACE (such as Lindat). Can we find a common fix?
For the I/O - there is no specific element in the web form for such info in CLARIN-DSPACE. We can add this to the description?

**Comment by jakoble** [comment]
Good point; it seems that those VLO entries that have the I/O are those that are harvested from non-DSPACE repos.

Yes, including the I/O in the description would therefore be a good idea (and richer descriptions about the tool's features are a good idea in any case :) ).
